edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -445,6 +448,15 @@ Scale1DQueryPageController = Class.create(QueryPageController, {
   }
 });
 
+Multiple2DScaleQueryPageController = Class.create(QueryPageController, {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add the "let", "const" or "var" keyword to this declaration of "Multiple2DScaleQueryPageController" to make it explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS2703)